NEWS.md

```diff
@@ -12,6 +12,7 @@
  * [MODORDERS-45](https://issues.folio.org/browse/MODORDERS-45) 
  * [MODORDERS-37](https://issues.folio.org/browse/MODORDERS-37)
  * [MODORDERS-34](https://issues.folio.org/browse/MODORDERS-34)
+ * [MODORDERS-33](https://issues.folio.org/browse/MODORDERS-33)
```

Comment:
Please move this from  `## 1.0.2` section because this is already released

Reply:
fixed